mappings/com/mojang/blaze3d/platform/GlStateManager.mapping

```diff
@@ -1,4 +1,84 @@
-CLASS com/mojang/blaze3d/platform/GlStateManager
+CLASS com/mojang/blaze3d/platform/GlStateManager com/mojang/blaze3d/platform/GlStateManager
+	METHOD alphaFunc (IF)V
+		ARG 0 func
+		ARG 1 ref
+	METHOD bindTexture (I)V
+		ARG 0 texture
+	METHOD blendEquation (I)V
+		ARG 0 mode
+	METHOD blendFunc (II)V
+		ARG 0 sfactor
+		ARG 1 dfactor
+	METHOD blendFuncSeparate (IIII)V
+		ARG 0 sFactorRGB
+		ARG 1 dFactorRGB
+		ARG 2 sFactorAlpha
+		ARG 3 dFactorAlpha
+	METHOD color3f (FFF)V
+		ARG 0 red
+		ARG 1 green
+		ARG 2 blue
+	METHOD color4f (FFFF)V
+		ARG 0 red
+		ARG 1 green
+		ARG 2 blue
+		ARG 3 alpha
+	METHOD colorMaterial (II)V
+		ARG 0 face
+		ARG 1 mode
+	METHOD depthFunc (I)V
+		ARG 0 func
+	METHOD depthMask (Z)V
+		ARG 0 mask
+	METHOD drawArrays (III)V
+		ARG 0 mode
+		ARG 1 first
+		ARG 2 count
+	METHOD light (IILjava/nio/FloatBuffer;)V
+		ARG 0 light
+		ARG 1 pname
+		ARG 2 params
+	METHOD lightModel (ILjava/nio/FloatBuffer;)V
+		ARG 0 pname
+		ARG 1 params
+	METHOD normal3f (FFF)V
+		ARG 0 nx
+		ARG 1 ny
+		ARG 2 nz
+	METHOD pixelStore (II)V
+		ARG 0 pname
+		ARG 1 param
+	METHOD polygonOffset (FF)V
+		ARG 0 factor
+		ARG 1 units
+	METHOD pushMatrix pushMatrix ()V
+	METHOD rotatef (FFFF)V
+		ARG 0 angle
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD setupSolidRenderingTextureCombine (I)V
+		ARG 0 color
+	METHOD texImage2D (IIIIIIIILjava/nio/IntBuffer;)V
+		ARG 0 target
+		ARG 1 level
+		ARG 2 internalFormat
+		ARG 3 width
+		ARG 4 height
+		ARG 5 border
+		ARG 6 format
+		ARG 7 type
+		ARG 8 pixels
+	METHOD texSubImage2D (IIIIIIIIJ)V
+		ARG 0 target
+		ARG 1 level
+		ARG 2 xOffset
+		ARG 3 yOffset
+		ARG 4 width
+		ARG 5 height
+		ARG 6 format
+		ARG 7 type
+		ARG 8 pixels
 	CLASS class_1016 AlphaTestState
```

Comment:
Why were all the methods moved above the inner classes?

Reply:
IDEA automatic pre-commit formatting.
Why? Did it mess something up?

Reply:
Hm? I'm asking why the methods of this class, which were below the inner classes, were moved to be above them. Why would IDEA format `.mapping` files?

Reply:
Don't know. But that is the only explanation I can come up with. I certainly didn't move it around myself, and I did have auto-formatting turned on.

Reply:
Can I resolve this? It seems like a non-issue, but I'm not sure.

Reply:
Yeah not an issue, the order was just wrong before